addons/http_routing/models/ir_http.py

```diff
@@ -302,11 +302,11 @@ def get_nearest_lang(cls, lang_code):
         """ Try to find a similar lang. Eg: fr_BE and fr_FR
             :param lang_code: the lang `code` (en_US)
         """
+        if not lang_code or lang_code in request.env['ir.module.module']._installed().keys():
+            return False
         short_match = False
-        if not lang_code:
-            return short_match
         short = lang_code.partition('_')[0]
-        for code in [code for code, _, _ in request.env['res.lang'].get_available()]:
+        for code in [code for code, _ in request.env['res.lang'].get_installed()]:
```

Comment:
Any reason for the double loop?
```suggestion
        for (code, _) in request.env['res.lang'].get_installed():
```

Reply:
Nop, applied your change